testcafe/tests/drive/classification_scenario.js

```diff
@@ -371,13 +391,21 @@ test(`${TEST_EMPTY_TRASH}`, async t => {
     withMask: maskDeleteFolder
   })
 
+  //We cannot use checkToastAppearsAndDisappears here, as both toast might appears at the same time, or the 1st one may disappear before the second one shows up.
+  isExistingAndVisibile(
+    selectors.alertWrapper.withText(
+      'Your trash is being emptied. This might take a few moments.'
+    ),
+    'Toast : Your trash is being emptied. This might take a few moments. '
+  )
+  isExistingAndVisibile(
+    selectors.alertWrapper.withText('The trash has been emptied.'),
+    'Toast : The trash has been emptied.'
+  )
+  //Wait for toast alert to disapear before taking screenshot
+  await t.wait(5000)
```

Comment:
await Promise.all([isExistingAndVisibile(1), isExistingAndVisibile(2)]) ? 

No need for wait 5000 ?

Reply:
```
await Promise.all([isExistingAndVisibile(1), isExistingAndVisibile(2)]) 
await Promise.all([t.expect( selectors.alertWrapper.withText('1....').exists)).notOk()),t..expect( selectors.alertWrapper.withText('.2...').exists)).notOk()))])
```

I still need to wait for the toast to disappear